runtime/doc/quickfix.txt

```diff
@@ -47,8 +47,23 @@ compiler (see |errorformat| below).
 Each quickfix list has a unique identifier called the quickfix ID and this
```

Comment:
Please add a new helptag `*quickfix-stack*` and then reference this from the `'chistory'` and `'lhistory'` options.